Jenkinsfile

```diff
@@ -329,13 +329,13 @@ pipeline {
                         pragmasToEnv()
                     }
                 }
-                stage('Determine Base Branch') {
+                stage('Determine Landing Branch') {
```

Comment:
"Landing" branch might be misleading too. If someone opens a PR to be merged into "foo", then "foo" is the immediate landing branch.

Reply:
Where `foo` is based on what?  Just so that we are using the same nomenclature, the branch against which a PR is create is called the _base branch_.  I'm specifically not looking for a base branch here.  For example, if somebody opens a PR on a branch called `foo` with a base branch of `master` and then opens a PR on a branch named `bar` against the branch `foo`, `foo` is the _base branch_ but `master` is the _landing branch_.  It's that latter, _landing branch_ name that I am looking for.

Reply:
Right. The devil is in the nuance :) I think "release" might be the most appropriate.